app.js

```diff
@@ -167,14 +167,11 @@ app.get('/healthcheck', function(req, res) {
   });
 });
 
+if (!env.get('WEBMAKER_REDIRECT_URL')) {
```

Comment:
let's just use this:
```
var landing = env.get("AUDIENCE") + "/goggles";
...
app.get("/", function(req, res) {
  res.redirect(302, landing);
});
```
no need for a new env. var in this case, we already know where webmaker.org lives